e2e/testcases/correction-birth/correct-birth-record-4.spec.ts

```diff
@@ -0,0 +1,1053 @@
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('father-view-group')).toBeTruthy()
+        expect(page.url().includes('#fatherPassport')).toBeTruthy()
+
+        await page.locator('#fatherPassport').fill(updatedFatherDetails.id)
+
+        await page.waitForTimeout(500)
+
+        await page.getByRole('button', { name: 'Back to review' }).click()
+
+        /*
+         * Expected result: should
+         * - redirect to review page
+         * - show previous id with strikethrough
+         * - show updated id
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('review')).toBeTruthy()
+
+        await expect(
+          page.locator('#father-content #ID').getByText(updatedFatherDetails.id)
+        ).toBeVisible()
+      })
+
+      test('4.4.1.6 Change usual place of residence', async () => {
+        await page
+          .locator('#father-content #Usual')
+          .getByRole('button', { name: 'Change', exact: true })
+          .click()
+
+        /*
+         * Expected result: should
+         * - redirect to father's details page
+         * - focus on father's Usual place of resiedence
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('father-view-group')).toBeTruthy()
+        expect(page.url().includes('#countryPrimary')).toBeTruthy()
+
+        await page.locator('#statePrimaryFather').click()
+        await page.getByText(updatedFatherDetails.address.province).click()
+
+        await page.locator('#districtPrimaryFather').click()
+        await page.getByText(updatedFatherDetails.address.district).click()
+
+        await page
+          .locator('#cityPrimaryFather')
+          .fill(updatedFatherDetails.address.town)
+
+        await page
+          .locator('#addressLine1UrbanOptionPrimaryFather')
+          .fill(updatedFatherDetails.address.residentialArea)
+
+        await page
+          .locator('#addressLine2UrbanOptionPrimaryFather')
+          .fill(updatedFatherDetails.address.street)
+
+        await page
+          .locator('#addressLine3UrbanOptionPrimaryFather')
+          .fill(updatedFatherDetails.address.number)
+
+        await page
+          .locator('#postalCodePrimaryFather')
+          .fill(updatedFatherDetails.address.zipCode)
+
+        await page.waitForTimeout(500)
+
+        await page.getByRole('button', { name: 'Back to review' }).click()
+
+        /*
+         * Expected result: should
+         * - redirect to review page
+         * - show previous Usual place of resiedence with strikethrough
+         * - show updated Usual place of resiedence
+         */
+
+        expect(page.url().includes('correction')).toBeTruthy()
+        expect(page.url().includes('review')).toBeTruthy()
+
+        await expect(
+          page.locator('#father-content #Usual').getByRole('deletion').nth(1)
```

Comment:
Could use the same pattern as suggested in `e2e/testcases/death/declaration/death-declaration-11.spec.ts`